tls_codec/derive/src/lib.rs

```diff
@@ -424,7 +465,27 @@ fn impl_tls_size(parsed_ast: TlsStruct) -> TokenStream2 {
             generics,
             members,
             member_prefixes,
+            member_skips,
         }) => {
+            // Filter out skipped members.
+            let (members, member_prefixes) = {
+                let mut tmp_members: Vec<Member> = Vec::new();
+                let mut tmp_member_prefixes: Vec<Prefix> = Vec::new();
+
+                for ((member, prefix), skip) in members
+                    .into_iter()
```

Comment:
Using `iter` everywhere here and only cloning the `member` when needed should be more efficient.